src/Control/CookieJar.php

```diff
@@ -180,4 +187,17 @@ protected function outputCookie(
         }
         return false;
     }
+
+    /**
+     * Get the correct samesite value - Session cookies use a different configuration variable.
+     *
+     * @deprecated 5.0 The relevant methods will include a `$sameSite` parameter instead.
+     */
+    private function getSameSite(string $name): string
```

Comment:
This private function can be unit tested by using using reflection to change its visibility

Reply:
Test added